src/components/Spawn/SpawnAnimation.component.tsx

```diff
@@ -0,0 +1,53 @@
+import { useFrame, useGraph } from '@react-three/fiber';
+import React, { useMemo, useEffect, FC } from 'react';
+import { AnimationMixer, Group, LoopRepeat } from 'three';
+import { triggerCallback, useGltfLoader } from '../../services';
+import { SpawnState } from '../Avatar/Avatar.component';
+
+interface SpawnAnimationProps {
+  avatar: Group;
+  onSpawnAnimationFinish?: () => void;
+  onMountAnimation: SpawnState['onMountAnimation'];
+}
+
+export const SpawnAnimation: FC<SpawnAnimationProps> = ({ avatar, onSpawnAnimationFinish, onMountAnimation }) => {
+  const [animationRunning, setAnimationRunning] = React.useState(true);
+
+  useEffect(() => {
+    if (!animationRunning) {
+      triggerCallback(onSpawnAnimationFinish);
+    }
+  }, [onSpawnAnimationFinish, animationRunning]);
+
+  const { nodes: avatarNode } = useGraph(avatar);
+  const animationAvatar = useGltfLoader(onMountAnimation?.src || '');
```

Comment:
Animation source needs to be validated before trying to call it with a `Loader`.

We're using this method which handles glb input validation
```
isValidGlbFormat(animationSrc)
```

Reply:
done in the spawn component (root one)